client/client.go

```diff
@@ -445,6 +445,16 @@ const (
 	ErrorKindDaemonRestart = "daemon-restart"
 )
 
+// IsRetryableError returns true if the given error is an error
+// that can be retried later.
+func IsRetryableError(err error) bool {
```

Comment:
wonder if this should drop the "Error" to be more consistent with stdlib names like this, IsRetryable ?